E/Extrae/build_tarballs.jl

```diff
@@ -51,39 +56,53 @@ autoreconf -fvi
     --disable-openmp \
     --disable-nanos \
     --disable-smpss \
-    --without-mpi \
-    ${ENABLE_CUDA:+--with-cuda=$prefix/cuda} \
+    ${FLAGS_MPI:---without-mpi} \
+    ${FLAGS_CUDA:---without-cuda} \
     --with-binutils=$prefix \
     --with-unwind=$prefix \
-    --with-xml-prefix=$prefix \
+    --with-xml=$prefix \
     --with-papi=$prefix
 
 make -j${nproc}
 make install
 """
 
-non_cuda_platforms = [
+platforms = [
     Platform("x86_64", "linux"; libc="glibc"),
     Platform("aarch64", "linux"; libc="glibc"),
     Platform("powerpc64le", "linux"; libc="glibc"),
 ]
-non_cuda_platforms = expand_cxxstring_abis(non_cuda_platforms)
+platforms = expand_cxxstring_abis(platforms)
 # (Almost) Same as PAPI
 cuda_platforms = CUDA.supported_platforms(; max_version=v"12.2")
 cuda_platforms = expand_cxxstring_abis(cuda_platforms)
 
+mpi_platforms, mpi_dependencies = MPI.augment_platforms(platforms)
+filter!(platform -> platform["mpi"] != "mpitrampoline", mpi_platforms)
```

Comment:
Why is this removed?

Reply:
It's giving errors because MPITrampoline doesn't provide `libmpi.so`, which is what Extrae is looking for.

Maybe a symbolic link could work?

Reply:
No, we shouldn't make up fake symlinks which are going to be broken. Also, libblastrampoline isn't alone in not using libmpi as soname (also some vendor mpi libraries have different names), it sounds like extrae should allow optionally changing the library name.